CPT331.Data.Migration/Migrations/Crime.spGetCrime.sql

```diff
@@ -4,15 +4,16 @@ AS
 
 SELECT
 	ID,
-	LocalGovernment
-	Name,
+	LocalGovernmentAreaID,
```

Comment:
Fix up sproc. This will need to be changed in the future to support paging and that kind of stuff.